tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -748,14 +748,17 @@ public function test_process_directives_does_not_change_inner_html_in_math() {
 	 * @param string $directive_value The directive attribute value to evaluate.
 	 * @return mixed The result of the evaluate method.
 	 */
-	private function evaluate( $directive_value ) {
-		$generate_state = function ( $name ) {
+	private function evaluate( $directive_value, ) {
+		$obj            = new stdClass();
+		$obj->prop      = 'property';
+		$generate_state = function ( $name, $obj = null ) {
```

Comment:
It seems like this could be simplified and follow the same pattern as the rest of the state and statically set `$obj->property = $name` instead of taking an `$obj` parameter.